pkg/cli/admin/ocpcertificates/regeneratemco/updatesecrets.go

```diff
@@ -0,0 +1,105 @@
+package regeneratemco
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/vincent-petithory/dataurl"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+
+	"k8s.io/client-go/kubernetes"
+)
+
+func (o *RegenerateMCOOptions) RunUserDataUpdate(ctx context.Context) error {
+	return o.updateSecrets(ctx)
+}
+
+func (o *RegenerateMCOOptions) updateSecrets(ctx context.Context) error {
+	clientConfig, err := o.RESTClientGetter.ToRESTConfig()
+	if err != nil {
+		return err
+	}
+	clientset, err := kubernetes.NewForConfig(clientConfig)
+	if err != nil {
+		return err
+	}
```

Comment:
nit: if you had a `Complete()` method, you could store these in your `RegenerateMCOOptions` and would not have to reinitialize here in case of a call from `func (o *RegenerateMCOOptions) Run(ctx context.Context) error {`

It would also help you with unit testing where you'd be able to supply your fake clients to the method.